backend/tournesol/tests/test_api_video_rate_later.py

```diff
@@ -242,3 +243,57 @@ def test_authenticated_can_delete(self):
         )
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         self.assertEqual(user.videoratelaters.count(), 0)
+
+    def test_automatically_removed_after_4_comparisons(self):
```

Comment:
I think this should be part of the comparisons API's tests, because it's expected that `auto_remove_from_rate_later` will be called by the comparisons endpoints, not the rate-later ones.